code/API_definitions/qos-profiles.yaml

```diff
@@ -206,6 +206,14 @@ components:
           example: "QoS profile for video streaming"
         status:
           $ref: "#/components/schemas/QosProfileStatusEnum"
+        countryAvailability:
+          description: |
+            A list of countries, and optionally networks, for which the API provider makes the profile available.
+            - When profiles are filtered for a specific device (i.e. using a 3-legged access token or an explicit device filter parameter), the profile can be used by that device in any of these countries when connected to any of the listed networks, or the API provider's network if none are listed
+            - When there is no device filter, availability for a given country means that the profile will be available to some devices connected to the networks listed for that country (or the API provider's network if none are listed), but not necessarily available to all devices, and not necessarily available to the same device for all listed countries and networks
```

Comment:
In this case, it is not clear if the service is available in roaming mode in some cases, or only available to home devices in the listed countries/networks. In combination with the point above, it may be a bit misleading.

Reply:
The proposal is that, if no specific device is identified, then the API cannot be used to determine if it is possible for a device to use the QoS profile on all listed networks (through roaming). Maybe it can, maybe it can't, maybe it works for some devices but not others. But the API consumer will have to find out some other way, ideally by querying the API for a specific device.

Maybe you could propose some alternative text to make that clearer, always remembering that not all networks are mobile networks!